support-frontend/assets/pages/[countryGroupId]/guardianLightLanding/components/headerCards.tsx

```diff
@@ -88,21 +92,27 @@ export function HeaderCards({ geoId }: HeaderCardsProps): JSX.Element {
 		];
 	const formattedPrice = simpleFormatAmount(currency, price);
 
-	const card1UrlParams = new URLSearchParams({
+	const guardianLightParams = {
 		product: 'GuardianLight',
 		ratePlan: contributionType,
 		contribution: price.toString(),
-	});
+	};
+	const card1UrlParams = new URLSearchParams(
+		returnLink
+			? {
+					...guardianLightParams,
```

Comment:
This isn't a change in this PR, but since we're touching this logic, is it right that `contribution` is in the URL params?

Reply:
Agree, `contribution` really applies to `support` where you can overpay. I think the `guardianLight` product you can't overpay. Have removed the `contribution`.